source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -239,7 +243,7 @@ void AuthenticatorImpl::onDestroy() {
 
 // Verify with a specific public key.
 void AuthenticatorImpl::verifyKey() {
-  const Status status = ::google::jwt_verify::verifyJwt(*jwt_, *jwks_data_->getJwksObj());
+  const Status status = ::google::jwt_verify::verifyJwt(*jwt_, *jwks_data_->getJwksObj(), now - jwt_exp_slack);
```

Comment:
We don't need to double check the exp an bnf here.  My suggestion is:
* change code in https://github.com/google/jwt_verify_lib/blob/master/src/verify.cc
* add a new function called verifyJwtWithoutTimeChecking(),  this function will not check nbf_ and exp_

Reply:
Err. Are you sure? jwt_verify_lib isn't an Istio thing, is it?